src/services/produce_api/produceAPI/requirements.txt

```diff
@@ -0,0 +1,51 @@
+argcomplete==1.11.1
```

Comment:
change this file to lis only 

Django
django-rest-swagger
djangorestframework
Pillow